### PR TITLE
Round vertical coordinate to closest legal flight level upon insertio…

### LINKED ISSUE #1976: inserting a point in sideview should use flight levels

### PR DESCRIPTION
**Pull request by joernu76**
Round vertical coordinate to closest legal flight level upon insertion in side view.

Fix #1976

**Purpose of PR?**:

Fixes #1976

**Does this PR introduce a breaking change?**
no

### CODE REVIEW

File: mslib/msui/mpl_pathinteractor.py [hunk 1]
```diff
@@ -977,6 +977,7 @@ def button_release_insert_callback(self, event):
         y = event.ydata
         wpm = self.waypoints_model
         flightlevel = float(pressure2flightlevel(y * units.Pa).magnitude)
+        flightlevel = 5.0 * round(flightlevel / 5)
```
Comment (ReimarBauer) [inline mslib/msui/mpl_pathinteractor.py:981]: I am not sure if I understand this again in a year , or what happen when someone refactors this

would you please so kind and add a comment above this line.

**Comment by ReimarBauer** [comment]
thx, just one question, we likly will have a 8.3.0 soon because of  that KML fix. Do you want this in stable?

**Comment by ReimarBauer** [comment]
@joernu76 for stable?

**Comment by joernu76** [comment]
This does not reaally fix a bug. -> develop

**Comment by ReimarBauer** [comment]
We are preparing a minor release. This means we can add features which don't need an API change.

Patch level (3rd number) would be Bug fixes only.

**Review by ReimarBauer — APPROVED** [review]

**Pull request merged**